perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -1341,9 +1350,9 @@ public void run() {
 				} catch(Exception ex) {
 					log.info("validateMemberAsync failed. Cause: {}", ex);
 					try {
-						getPerunBl().getAuditer().log(sess, "Validation of {} failed. He stays in {} state.", member, oldStatus);
+						getPerunBl().getAuditer().log(sess, new MemberValidatedFailed(member, oldStatus));
 						log.info("Validation of {} failed. He stays in {} state.", member, oldStatus);
-					} catch(InternalErrorException internalError) {
+					} catch(RuntimeException internalError) {
```

Comment:
The previous version of `auditer.log()`  method had in its declaration that it throws InternalErrorException but in fact there was no way it could be thrown. So, I have changed it to RuntimeException because I was not sure if the log in this `catch` clause is important.

Reply:
Ok, but in this case you will suppress any thrown Runtime exception. I am not sure, if it is ok this way. I would remove whole try-catch block and also the log itself. If Internal error can't be thrown, it is useless.

Reply:
Ok, I will remove it.